src/argilla/server/schemas/v1/questions.py

```diff
@@ -86,3 +90,21 @@ class Question(BaseModel):
 
     class Config:
         orm_mode = True
+
+
+class TextQuestionSettingsUpdate(UpdateSchema):
+    use_markdown: Optional[bool]
+
+    __non_explicit_none__ = {"use_markdown"}
+
+
+class LabelSelectionSettingsUpdate(UpdateSchema):
+    visible_options: Optional[int]
+
+
+class QuestionUpdate(UpdateSchema):
+    title: Optional[str]
+    description: Optional[str]
+    settings: Optional[Union[Dict[str, Any], TextQuestionSettingsUpdate, LabelSelectionSettingsUpdate]]
+
+    __non_explicit_none__ = {"title", "description", "settings"}
```

Comment:
Description can be nullable in DB. For me, it makes sense also provide this option from the update endpoint. 
```suggestion
    __non_explicit_none__ = {"title", "settings"}
```